app/assets/stylesheets/base.scss

```diff
@@ -56,12 +58,9 @@ body > footer p {
 }
 
 .section-ellipsis,
-.section-you {
-  color: rgb(93, 32, 38);
-}
-
+.section-you,
 .section-sponsors {
-  background-color: rgb(116, 39, 47);
+  color: rgb(93, 32, 38);
```

Comment:
Perhaps out of scope for this PR, but it might be good to start using a few variables for colors? E.g. `var(--hero-background-color)`?

Reply:
this could be another PR but I would prefer that we first define the stack for the new design (TW or not basically)